src/browser/rvm/rvm_message_bus.ts

```diff
@@ -352,6 +356,26 @@ export class RVMMessageBus extends EventEmitter  {
             }
         }
     }
+
+    /**
+     * Retrieves information about a plugin
+     */
+    public getPluginInfo(manifestUrl: string, opts: Plugin): Promise<PluginQueryResponse> {
```

Comment:
goes this `getPluginInfo` or `requestPluginInfo`?

Reply:
I can certainly change to `requestPluginInfo`, no problem... what's the main reason for change?

Reply:
just semantics, it does not return or get anything, just posts a message. Presumably the RVM will answer with what we need on a message we are subscribed to somewhere else